content/news/2024.pml

```diff
@@ -54,6 +54,32 @@ Older News: <%=reflist%>
 </div>
 -->
 
+
+<!----------------------------------------------------------->
+<div class="news-item tag-link" id="summit-2024-cfp">
+  <a href="#summit-2024-cfp"><span class="tag-span"># </span></a>
+  19 February 2024: <a href="../Foundation/Summit/2024/cfp">Call For Presentations</a> for the <a href="../Foundation/Summit/2024">seL4 Summit 2024</a>
+</div>
+<div>
+  <a href = "../Foundation/Summit/2024">
+    <img src="/images/summit/sel4_summit_2024.svg"
+    style="width: 15%; float:right"
+    alt="seL4 summit" />
+  </a>
+  <h3 class="highlight">
+   <a href="../Foundation/Summit/2024/cfp">Call For Presentations</a> for the <a href="../Foundation/Summit/2024">seL4 Summit 2024</a>
+  </h3>
+  <ul>
+  <li>Share your seL4 work</li>
+  <li>Share your seL4 experience</li>
+  <li>Share your seL4 thoughts</li>
+  </ul>
+  <p>
+    Check the full <a href="../Foundation/Summit/2024/cfp">Call For Presentations</a>. To propose a talk, upload an abstract of one page or less by <strong>10 April 2023</strong> to the <strong><a href="../Foundation/Summit/2023/submit">submission portal</a></strong>.
+  </p>
```

Comment:
I would add: "Also note the [open invitation for a 5-minute slot to talk about your seL4 deployment]" (with a link to that section on the CfP.

Reply:
the link cfp#sel4-deployment doesn't work on the preview (but maybe it's a preview issue)